lib/type/any.ex

```diff
@@ -1,20 +1,34 @@
 defmodule Litmus.Type.Any do
-  @moduledoc """
-  Schema and validation for Any data type.
-  """
+  @moduledoc false
 
   defstruct required: false
 
   @type t :: %__MODULE__{
           required: boolean
         }
-end
 
-defimpl Litmus.Type, for: Litmus.Type.Any do
-  alias Litmus.Type
+  @spec validate_keys(map, binary, t) :: {:ok, map} | {:error, String.t()}
+  def validate_keys(data, field, type), do: add_required_errors(data, field, type)
```

Comment:
since this function only validates a single field, how about naming it `validate_field` instead?